app/assets/javascripts/discourse/app/components/sidebar/sidebar-section-form-link.js

```diff
@@ -0,0 +1,62 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+
+export default class SidebarSectionFormLink extends Component {
+  @tracked dragCssClass;
+
+  constructor() {
+    super(...arguments);
+    this.link = this.args.link;
+    this.reorderCallback = this.args.reorderCallback;
+    this.deleteLink = this.args.deleteLink;
+  }
```

Comment:
```suggestion
```

I don't think this constructor is needed at all since the properties are already accessible within the component from the `this.args` property.